app/src/main/java/ru/qveex/rst_tur/presentation/components/BottomNav.kt

```diff
@@ -45,7 +47,7 @@ fun RowScope.AddItem(
         icon = {
             Icon(
                 imageVector = screen.icon,
-                contentDescription = "Navigation Icon"
+                contentDescription = stringResource(R.string.navigation_icon_content_description)
```

Comment:
```suggestion
                contentDescription = screen.title
```
In this case you can use title of screen in bottom nav
But in `navigation
/Screen.kt` strings still not localized